compose-builder/add-security.yml

```diff
@@ -106,6 +102,28 @@ services:
       - consul
       - vault
 
+  security-bootstrap-database:
+    image: ${CORE_EDGEX_REPOSITORY}/docker-edgex-security-bootstrap-redis-go${ARCH}:${CORE_EDGEX_VERSION}${DEV}
```

Comment:
Image name is wrong. Just confirmed `nexus3.edgexfoundry.org:10004/docker-security-bootstrap-redis-go:master` was created.

change `docker-edgex-security-bootstrap-redis-go` to `docker-security-bootstrap-redis-go:master`

Reply:
I'll push before 8p PT.

Reply:
Done